v1/index.mjs

```diff
@@ -14,6 +14,7 @@ const { graphqlKoa, graphiqlKoa } = apollo
 const api = new Router({prefix: '/v1'})
 
 // food
+api.get('/food/menu/pause', pauseMenu)
```

Comment:
Should this be `/food/pause/menu`? Since it's a different setup?

Reply:
Is that what you want? It makes sense to me the way it is, but I'm indifferent.

Reply:
I would rather have the pause have its own sub-namespace under `/food`, yes.